src/fsd/4-entities/npc/npcService.ts

```
@@ -0,0 +1,119 @@
+                return Alliance.Xenos;
+            case 'Xenos':
+                return Alliance.Xenos;
+            case 'Chaos':
+                return Alliance.Chaos;
+            case 'Imperial':
+                return Alliance.Imperial;
+            default:
+                return undefined;
+        }
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)